docs/pages/access-controls/access-request-plugins/servicenow.mdx

```diff
@@ -76,6 +76,10 @@ to Management -> Integrations -> Enroll New Integration.
 
 The plugin requires credentials that can read and write to the incident table.
 
+The 'Close Code' specified here will be the one used when resolving ServiceNow
+incidents. The 'Caller' will be the username of the requesting Teleport user.
+(The 'Caller' field will only be filled if there is an existing ServiceNow user of the same name)
```

Comment:
```suggestion
The "Close Code" specified here will be the one used when resolving ServiceNow
incidents. The "Caller" will be the username of the requesting Teleport user.
(The "Caller" field will only be filled if there is an existing ServiceNow user of the same name.)
```

- We tend to use double quotes in the docs.
- Adding a period in the parentheses to fix grammar.